src/System.Private.CoreLib/src/System/Diagnostics/Contracts/Contract.CoreCLR.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
```

Comment:
Could you please remove these `**
===========================================================*/` style comments while you are at it?